libnmstate/nm/ovs.py

```diff
@@ -173,15 +173,16 @@ def _get_bridge_port_info(port_profile, devices_info):
         number_of_interfaces = len(port_slave_names)
         if number_of_interfaces == 1:
             port_info[OB.Port.NAME] = port_slave_names[0]
+            if vlan_mode:
```

Comment:
The vlan mode as seen from NM/ovsdb is not access/trunk, it is a bunch of other values. These need to be "translated".
Specifically:
access => access
trunk => trunk
native-untagged => trunk
// The below are rare, and will require an additional flag probably if requested.
// We should not support them at the moment.
native-tagged => trunk
dot1q-tunnel => trunk

Reply:
You want me to do that in this PR?

Reply:
If we don't differentiate the `trunk` and `native-untagged`, when user apply `trunk` mode, how should we know which one is user requested.

I intend to keep it as it was.

Reply:
I prefer the mode not to include this subset cases as OVSDB exposes but to use flags.
The `native-untagged` is covered through `enable-native` flag and `mode` is accepting only `trunk` and `access` in the schema (in sync with the Linux Bridge VLAN filter as well).

Reply:
What will happen when OVS introduce a new mode?
We raise exception?

Reply:
Good point. We can add to the schema an `unknown` option.
But IMO, any new option may be classified as trunk and we can print a a warning/error that it is not a supported mode in nmstate.